### PR TITLE
Updates to Dynamic ISF Plugin

### PR DESCRIPTION
**Pull request by tim2000s**
Adjustment factor for Dynamic ISF added to allow modification of the ISF outcome;
Austosens based on Rolling 24 hour TDD/7-day Average TDD in Determine Basal to replace Autosens outputs; is used to modify basal rate and to adjust target when Autosens switch is enabled (for testing before rejigging profile and Autosens components).
When glucose levels are flat and eventual BG is predicted below current bg, use an average of current and expected BG to determine dosing ISF.

### CODE REVIEW

File: app/src/main/java/info/nightscout/androidaps/plugins/aps/openAPSSMBDynamicISF/DetermineBasalAdapterSMBDynamicISFJS.kt [hunk 1]
```diff
@@ -194,9 +194,9 @@ class DetermineBasalAdapterSMBDynamicISFJS internal constructor(private val scri
         this.profile.put("current_basal_safety_multiplier", sp.getDouble(R.string.key_openapsama_current_basal_safety_multiplier, 4.0))
 
         //mProfile.put("high_temptarget_raises_sensitivity", SP.getBoolean(R.string.key_high_temptarget_raises_sensitivity, SMBDefaults.high_temptarget_raises_sensitivity));
-        this.profile.put("high_temptarget_raises_sensitivity", false)
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/aps/openAPSSMBDynamicISF/DetermineBasalAdapterSMBDynamicISFJS.kt:197]: @AdrianLxM do you remember why this was always false?

**Comment by tim2000s** [comment]
If I remember correctly, it was because it caused issues with regular profile switching resulting in very small ISF numbers. 

I think there's an additional change in the code required to fully enable it though, as in this push, the switches in preferences remain greyed out.

**Comment by MilosKozak** [comment]
what happened there is so many commits?

**Comment by MilosKozak** [comment]
@tim2000s could you squash these commits?

**Pull request closed without merging**